docs/001_develop/03_client-capabilities/006_charts/03_chart_html_api.mdx

```diff
@@ -290,7 +290,7 @@ Property and attribute binding examples for Genesis Component syntax. Closing ta
         <tr>
             <td>orderBy</td>
             <td><code>string</code></td>
-            <td>This option can be used to select a Data Server index (defined in the Data Server query), which is especially useful if you want the data to be sorted in a specific way.</td>
+            <td>This option can be used to select a [Data Server index](../../../server-capabilities/real-time-queries-data-server/#indices) (defined in the Data Server query), which is especially useful if you want the data to be sorted in a specific way.</td>
```

Comment:
Please don't use relative links. Change to `(/develop/server-capabilities/real-time-queries-data-server/#indices)`

Reply:
Updated